python/ray/autoscaler/command_runner.py

```diff
@@ -462,9 +467,7 @@ def _run_helper(self,
                     "Command failed: \n\n  {}\n".format(quoted_cmd)) \
                     from None
             else:
-                raise click.ClickException(
-                    "SSH command Failed. See above for the output from the"
-                    " failure.") from None
+                raise click.ClickException("SSH command failed") from None
```

Comment:
Can we add a check here for whether the output was redirected (`subprocess_output_util.is_output_redirected`)? Because sometimes the output will actually be there and we would want to keep the old error message.